lsm-tree.cabal

```diff
@@ -34,7 +34,14 @@ common wno-x-partial
 common language
   default-language:   GHC2021
   default-extensions:
+    DeriveAnyClass
+    DerivingStrategies
+    DerivingVia
+    ExplicitNamespaces
     GADTs
+    LambdaCase
+    MultiWayIf
```

Comment:
I'm not especially keen on encouraging this one. It looks like it's used in only one place at the moment.

Reply:
It's either one line here or there. Either we use it, or not use it at all.

Reply:
You're quite right. I've added a patch.